### ISSUE TITLE
update release binary version

### ISSUE THREAD

**Issue by Arvin2015** [huaweicloud/packer-plugin-huaweicloud#113]
encounter error while install local plugin

Error: Invalid version

Packer can only install plugin releases with this command (ex: 1.0.0) or
development pre-releases (ex: 1.0.0-dev), the binary's reported version is
"0.4.0-rc1"

**Comment by Arvin2015** [comment]
duplicate issue

**Comment by Arvin2015** [comment]
duplicate issue, noticed that 1.2.1 fixed this problem and released 1 hour ago.

**Comment by Zippo-Wang** [comment]
Yes, you can use [v1.2.1](https://github.com/huaweicloud/packer-plugin-huaweicloud/releases/tag/v1.2.1) and retry your script

**Comment by huaweiclouddev** [comment]
## 🤖 Stale Bot

🔒 因 35 天无活动自动关闭。如需重新打开请评论。

<sub>集中管理 · 详见 [治理规范](https://github.com/huaweicloud/.github/blob/main/GOVERNANCE.md)</sub>
